.prettierignore

```diff
@@ -32,3 +32,4 @@ CHANGELOG.md
 
 # File generated by @hive/app#generate-changelog script
 packages/web/app/src/components/ui/changelog/generated-changelog.ts
+packages/web/docs/src/pages/docs/self-hosting/changelog.mdx
```

Comment:
I tried to use a `.tsx` file, but nextra did not work with `RemoteContent` unless the file uses the `.mdx` extension. I could not find any information in the extra documentation about that topic. So I switched to `.mdx`, even though now we don't get TypeScript benefits, I also had to disable prettier for the file, as it would destroy it every time it tries to format it.